src/_P183_modbus.ino

```diff
@@ -243,12 +203,17 @@ boolean Plugin_183(uint8_t function, struct EventStruct *event, String& string)
     case PLUGIN_READ:
     {
       P183_data_struct *P183_data = static_cast<P183_data_struct *>(getPluginTaskData(event->TaskIndex));
+
+      if (P183_data == nullptr) {
+        addLogMove(LOG_LEVEL_INFO, F("******* Modbus: Read invalid data struct"));
+        return false;
```

Comment:
Yes, it's helpful during development to have recognizable markers in the log, but a) this could be an ERROR log, and b) no need to add the `***` as they are often forgotten to remove once code is becoming final...